cluster/charts/rook-ceph/values.yaml

```diff
@@ -67,6 +67,8 @@ csi:
   # CSI Rbd plugin daemonset update strategy, supported values are OnDelete and RollingUpdate.
   # Default value is RollingUpdate.
   #cephFSPluginUpdateStrategy: OnDelete
+  # Allow starting unsupported ceph-csi images
+  #allowUnsupportedImage: true
```

Comment:
What about naming this setting `allowUnsupportedVersion`?

Reply:
Will change that